### PR TITLE
Phoenix v1/correct fee adjustment overflow

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: tests/test_phoenix.rs [hunk 1]
```diff
@@ -3113,15 +3113,15 @@ async fn test_phoenix_place_multiple_memory_management() {
         ..
     } = &phoenix_ctx;
     // Send 40 orders on each side to verify there is enough compute to do so
-    let bids = (1..41)
+    let bids = (1..30)
```
Comment (eugene-chen) [inline tests/test_phoenix.rs:3116]: update comment

File: tests/test_phoenix.rs [hunk 2]
```diff
@@ -2849,15 +2849,15 @@ async fn test_phoenix_place_multiple_limit_orders() {
         .unwrap();
 
     // Send 100 orders on each side to verify there is enough compute to do so
-    let bids = (1..101)
+    let bids = (1..30)
```
[the remark this answers is not in the document]
Comment (eugene-chen) [reply tests/test_phoenix.rs:2852]: can we check the return data here?

[the remark this answers is not in the document]
Comment (jarry-xiao) [reply tests/test_phoenix.rs:2852]: I think it's difficult to check return data with program test. I was planning to take a slightly closer look at that interface

File: tests/test_phoenix.rs (hunk 1 above)
Comment (eugene-chen) [reply tests/test_phoenix.rs:3116]: same above where it used to be 100

**Review by eugene-chen — APPROVED** [review]

**Pull request merged**